tests/token/erc20/test_ERC20.py

```diff
@@ -51,4 +51,4 @@ def contract_factory(contract_classes, erc20_init):
 
 
 class TestERC20(ERC20Base):
-    pass
\ No newline at end of file
+    pass
```

Comment:
```suggestion
    pass
    
```

Reply:
I think the end of line is there, but I got confused by GitHub UI around this somehow?